src/main/resources/story/walmart/SearchForProductBySearchBox.story

```diff
@@ -0,0 +1,22 @@
+Description: Story to test product search by Searchbox;
+
+Scenario: Open Walmart main page
+Given I am on a page with the URL 'https://www.walmart.com/'
+Then the page title is 'Walmart.com | Save Money. Live Better.'
+
+
+Scenario: Search product by Searchbox
+When I initialize the scenario variable `searchLine` with value `Playhouse`
+When I enter '${searchLine}' in a field by the xpath '//input[@id='global-search-input']'
+When I click on an element by the xpath '//button[@id='global-search-submit']'
+!-- Check that at least one product found
+Then the number of elements found by the xpath '//div[@class='search-result-listview-item Grid']//span[contains(., '${searchLine}')] | span[contains(., #{toLowerCase("${searchLine}")})]' is greater than '0'
```

Comment:
I think here it would be easier just to use 'Examples' functionality instead of initializing the variable.

For example:
When I enter '<searchLine>' in a field by the xpath '//input[@id='global-search-input']
Then the number of elements found by the xpath '//div[@class='search-result-listview-item Grid']//span[contains(., '<searchLine>')] | span[contains(., #{toLowerCase("<searchLine>")})]' is greater than '0'
 Examples:
|searchLine|
| Playhouse |

Reply:
In this step :

I would recommend to use this xpath ///div[@class='search-result-listview-item Grid']//child::*[contains(@title, '<searchLine>')] and comparison is 'less than or equal to' '20' as it will at the same time check the pagination

Reply:
In this step I would recommend to use this xpath ///div[@class='search-result-listview-item Grid']//child::*[contains(@title, '\<searchLine\>')] and comparison is 'less than or equal to' '20' as it will at the same time check the pagination

Reply:
Added checks for capitalized and lower-cased `'<searchLine>'` in @title as well:
`//div[@class='search-result-listview-item Grid']//child::*[contains(@title, #{capitalize('<searchLine>')}) or contains(@title, #{toLowerCase('<searchLine>')})]`